src/main/java/no/nav/foreldrepenger/oversikt/innhenting/inntektsmelding/FpOversiktInntektsmeldingDto.java

```diff
@@ -0,0 +1,37 @@
+package no.nav.foreldrepenger.oversikt.innhenting.inntektsmelding;
+
+import java.math.BigDecimal;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public record FpOversiktInntektsmeldingDto(
```

Comment:
formatering

Reply:
Jeg sliter med formatteringer. Når jeg gjør autoformat så får jeg det på dette formatet.

```

public record InntektsmeldingDto(int versjon, boolean erAktiv, BigDecimal stillingsprosent, BigDecimal inntektPrMnd, BigDecimal refusjonPrMnd,
                                 String arbeidsgiverNavn, String journalpostId, String kontaktpersonNavn, String kontaktpersonNummer,
                                 LocalDateTime innsendingstidspunkt, LocalDateTime mottattTidspunkt, LocalDate startDatoPermisjon,
                                 List<NaturalYtelse> bortfalteNaturalytelser, List<Refusjon> refusjonsperioder) {
    public record NaturalYtelse(LocalDate fomDato, LocalDate tomDato, BigDecimal beloepPerMnd, String type) {
    }

    public record Refusjon(BigDecimal refusjonsbeløpMnd, LocalDate fomDato) {
    }
}
```

Som jeg antar ikke er noe bedre. Tips til hvordan jeg får fin formattering? Er det sjekket inn en editorconfig jeg ikke bruker eller no?

Reply:
tror det er sånn .editorconfig'n vår er nå, dårlig på records. Så regner med at den fungerer for deg

Reply:
Pleier dere da la den være sånn, eller manuelt newline'e?